documentation/binding-data/tutorial-flow-components-binder-beans.asciidoc

```diff
@@ -2,25 +2,19 @@
 title: Binding Beans to Forms
 order: 4
 layout: page
-subnav_auto_list_numbers: true
-subnav:
```

Comment:
@pleku @tmattsso I think we maybe already discussed this, but please refresh my bad memory. Are these subnav things still used somewhere or not?

Reply:
unless @pleku objects, it can be removed. No one seems to have any idea what those things do.